src/index.js

```diff
@@ -1,4 +1,4 @@
-import bindings from "bindings";
+const bindings = require("bindings");
```

Comment:
Isn't this the wrong way around? I keep getting that error and fixed it by changing it back to `import bindings from "bindings";`

Reply:
You're right, sorry! This PR was originally to fix the fact that the code was improperly using 'import' when it should have been using 'require'. Then we decided to move the whole thing to ESM and I forgot to revert that file.

Reply:
No worries! Glad it works now. Thanks for maintaining this repo.